k8s/prod/common/bsp/bulk-scan-orchestrator.yaml

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: bulk-scan-orchestrator
+  namespace: bsp
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: bulk-scan-orchestrator
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: bulk-scan-orchestrator
+    version: 0.2.0
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/bulk-scan/orchestrator:latest
+      environment:
+        CCD_LOG_LEVEL: none
```

Comment:
this correct? not INFO?

Reply:
this is sth custom, none correct.

Reply:
This was introduced only for testing reasons. We should remove this setting and all custom integration around it

Reply:
default is full , you mean default should be none ?

Reply:
I mean this config should go away completely: from here and orchestrator service itself. Doesn't block this release, but definitely not needed anymore

Reply:
removed from application and here